src/main/java/org/spongepowered/plugin/meta/SpongeExtension.java

```diff
@@ -0,0 +1,15 @@
+package org.spongepowered.plugin.meta;
+
+public final class SpongeExtension {
+
+    public String assets; // public for mixin compatibility
+
+    public SpongeExtension(String assets) {
```

Comment:
I think you can just use a no-arg constructor here, I'd only use a constructor for required things